src/foundry/foundry.js/clientDocumentMixin.d.ts

```diff
@@ -157,7 +157,7 @@ export declare class ClientDocumentMixin<T extends foundry.abstract.Document<any
    * @param options - Sorting options provided to SortingHelper.performIntegerSort
    * @returns The Document after it has been re-sorted
    */
-  sortRelative(options: SortOptions<this>): Promise<this>;
+  sortRelative(options: any): Promise<any>;
```

Comment:
Putting in `: any` because I haven't had time to figure out where the type mismatches are coming from yet.